frontend/pulse3d/src/pages/upload-form.js

```diff
@@ -433,7 +433,7 @@ export default function UploadForm() {
     <Container>
       <Uploads>
         <Header>Run Analysis</Header>
-        {tabSelection === "0" ? (
+        {tabSelection === "Analyze New Files" ? (
```

Comment:
Should this be the same as the check that got changed above? They were different before

Reply:
the difference in indices is what's causing the issue you're seeing. Only one of them was changed in the PR that swaps the options